agent/agents/process/process_logger.go

```diff
@@ -116,6 +117,27 @@ func replacer(redactWords []string) *strings.Replacer {
 	return strings.NewReplacer(r...)
 }
 
+var matchLogLevelRegex = regexp.MustCompile(`level=(\w+)`)
```

Comment:
Ok, I read your comment about benchmark, and looks like using regex moves performance down. So can we extract it by using a strings package instead of regex? Something between your first implementation and this one.

Reply:
@BupycHuk regex has fine performance;

With do logging by `logrus.StandardLogger().Logln(logrus.InfoLevel, line)`

```text
BenchmarkMatchLevelByStringHasPrefix-12    	  190632	      7718 ns/op	     502 B/op	      11 allocs/op
BenchmarkMatchLevelByRegex-12              	  151029	      8167 ns/op	     430 B/op	      11 allocs/op
```